migtests/tests/pg/multiple-schemas/validate

```diff
@@ -145,16 +146,27 @@ def migration_completed_checks(tgt):
 			"orafce", "pageinspect", "pg_buffercache", "pg_cron", "pg_freespacemap", "pg_hint_plan", "pg_prewarm", "pg_stat_monitor",
 			"pg_stat_statements", "pg_trgm", "pg_visibility", "pgaudit", "pgcrypto", "pgrowlocks", "pgstattuple", "plpgsql",
 			"postgres_fdw", "refint", "seg", "sslinfo", "tablefunc", "tcn", "timetravel", "tsm_system_rows",
-			"tsm_system_time", "unaccent", "uuid-ossp", "yb_pg_metrics", "yb_test_extension" ]
+			"tsm_system_time", "unaccent", "uuid-ossp", "yb_pg_metrics", "yb_test_extension", "vector" ]
 		extensions = tgt.get_available_extensions()
 		print(f"expected extensions: {expectedExtensions}")
 		print(f"reported extensions: {extensions}")
 		tgtVersion = tgt.get_target_version()
 		print(f"target version: {tgtVersion}")
-		if "2.16" in tgtVersion:
-			expectedExtensions.remove("hll")
+		if "2.25" in tgtVersion:
```

Comment:
shouldn't we also enhance our unsupported extension issue reporting in analyze-schema/assess-migration? 

cc: @sanyamsinghal  i remember you already created a ticket for this. can you confirm?

Reply:
Yes
https://yugabyte.atlassian.net/browse/DB-15825